packages/docs/src/routes/qwikcity/routing/data/index.mdx

```diff
@@ -50,8 +50,7 @@ An endpoint `onGet` function retrieves data and makes it available to the compon
 
 ```typescript
 import { Resource, component$, useStore } from '@builder.io/qwik';
-import type { RequestHandler } from '@builder.io/qwik-city';
-import { useEndpoint } from "@builder.io/qwik-city";
+import type { RequestHandler, useEndpoint } from '@builder.io/qwik-city';
```

Comment:
This is a type import, byt useEndpoint() is a value. Unfortunately this is not correct and it will not compile.